galleries/plot_types/arrays/README.rst

```diff
@@ -1,6 +1,7 @@
 .. _array_plots:
 
-Plots of arrays and fields
---------------------------
+Gridded data: :math:`Z_{x, y}`
+-----------------------------
 
-Plotting for arrays of data ``Z(x, y)`` and fields ``U(x, y), V(x, y)``.
+Plots of arrays and images :math:`Z_{x, y}` and fields :math:`U_{x, y}, V_{x, y}`
+on `regular grids <https://en.wikipedia.org/wiki/Regular_grid>`_.
```

Comment:
I suggest to keep the index notation `` :math:`Z_{i, j}` `` here. All this naming is a bit suble but

1. Its confusing to use `Z_{x,y}` both for regular and and irregular grids
2. the i,j indices hint more at the regular structure.

Reply:
> That the irregular and regular gridded data are structurally the same

You mean the visualizations look alike (and the data are semantically similar)? The data are structurally different.

for regular grids, you have 2D structures $Z_{i,j}$ (and optionally $X_{i,j}$, $Y_{i,j}$). For irregular grids, you have 1D structures $x_i$, $y_i$ (and optionally $z_i$).

I know, it's a hassle to be precise here 😄. 

---
Maybe actually putting all the indices indices helps?


#### Pairwise data 
Plots of pairwise $(x_i, y_i)$, tabular $(v_{i,0}, \ldots, v_{i,n})$ and functional $f(x_i) = y_i$ data.

#### Gridded data
Plots of arrays and images $Z_{i,j}$ and fields $U_{i,j}$, $V_{i,j}$  on [regular grids](https://en.wikipedia.org/wiki/Regular_grid), optionally, with coordinates $X_{i,j}$, $Y_{i,j}$.

#### Irrgularly grided data
Plots of unstructured coordinate grids $(x_i, y_i)$ and data $z_i$ on such grids and  2D functions $f(x_i, y_i) = z_i$.

#### 3D and volumetric data
Plots of three-dimensional $(x_i, y_i, z_i)$, surface $f(x_i, y_i) = z_i$, and volumetric $({x_i, y_i, z_i, v_i})$ data using the [mpl_toolkits.mplot3d](https://output.circle-artifacts.com/output/job/c03b3700-1b96-4c83-824e-35a4013188ad/artifacts/0/doc/build/html/api/toolkits/mplot3d.html#module-mpl_toolkits.mplot3d) library.